Core/include/Acts/TrackFitting/detail/VoidKalmanComponents.hpp

```diff
@@ -84,4 +84,21 @@ struct VoidOutlierFinder {
   }
 };
 
+/// @brief void smoothing logic
+struct VoidReverseFilteringLogic {
+  /// @brief Public call mimicking an outlier finder
+  ///
+  /// @tparam track_state_t Type of the track state
+  ///
+  /// @param trackState The trackState to investigate
+  ///
+  /// @return Whether to run filtering in reversed direction as smoothing or not
+  template <typename track_state_t>
+  constexpr bool operator()(const track_state_t& trackState,
+                            const bool& reversedFiltering) const {
```

Comment:
I don't think a bool has to be given by reference.

Reply:
The pointer inside the reference will be larger than the bool to copy, so... :)

Reply:
I think we don't need the boolean anymore. We can return `false` in this `VoidReverseFilteringLogic`.

Reply:
I removed the reversedFiltering from the ReverseFilteringLogic.